riff-raff/app/ci/CIBuild.scala

```diff
@@ -53,9 +53,14 @@ object CIBuild extends Logging {
   lazy val initialBuilds: Future[Seq[CIBuild]] = {
     implicit val ec = ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(50))
     log.logger.info(s"Found ${initialFiles.length} builds to parse")
-    (Future.traverse(initialFiles) { location =>
-      Future(S3Build.buildAt(location))
-    }).map(_.flatten)
+    (Future.traverse(initialFiles)(l => Future(retrieveBuild(l)))).map(_.flatten)
+  }
+
+  private def retrieveBuild(location: S3Object): Option[CIBuild] = {
+    import cats.syntax.either._
+    S3Build.buildAt(location)
+      .leftMap(e => log.error(s"Problem getting build definition from $location: $e"))
```

Comment:
I was thinking it might be cleaner to fork a separate Observable of errors, but this at least doesn't completely silently swallow errors.